docs/ecs_guide.md

```diff
@@ -239,22 +250,23 @@ To create an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_role
     ```bash
     aws ecs create-service \
         --service-name prefect-worker-service \
-        --cluster <your-ecs-cluster> \
+        --cluster <ecs-cluster> \
         --task-definition <task-definition-arn> \
         --launch-type FARGATE \
         --desired-count 1 \
-        --network-configuration "awsvpcConfiguration={subnets=[<your-subnet-ids>],securityGroups=[<your-security-group-ids>]}"
+        --network-configuration "awsvpcConfiguration={subnets=[<subnet-ids>],securityGroups=[<your-security-group-ids>],assignPublicIp='ENABLED'}"
     ```
 
-    - Replace `<your-ecs-cluster>` with the name of your ECS cluster. 
-    - Replace `<path-to-task-definition-file>` with the path to the JSON file you created in Step 2, `<your-subnet-ids>` with a comma-separated list of your VPC subnet IDs. Ensure that these subnets are aligned with the vpc specified on the work pool in step 1.
-    - Replace `<your-security-group-ids>` with a comma-separated list of your VPC security group IDs. 
+    - Replace `<ecs-cluster>` with the name of your ECS cluster.
     - Replace `<task-definition-arn>` with the ARN of the task definition you just registered.
+    - Replace `<subnet-ids>` with a comma-separated list of your VPC subnet IDs. Ensure that these subnets are aligned with the vpc specified on the work pool in step 1. You can view subnet ids with the following command: 
+    `aws ec2 describe-subnets --filter Name=<vpc-id>`
+    - Replace `<security-group-ids>` with a comma-separated list of your VPC security group IDs.
 
     !!! tip "Sanity check"
-        The work pool page in the Prefect UI allows you to check the health of your workers - make sure your new worker is live!
+        The work pool page in the Prefect UI allows you to check the health of your workers - make sure your new worker is live! Although keep in mind it can take ECS a few minuter for a service to go online.
```

Comment:
```suggestion
        The work pool page in the Prefect UI allows you to check the health of your workers - make sure your new worker is live! Note that it can take a few minutes for an ECS service to come online.
```